frontend/test/metabase/scenarios/dashboard/chained-filters.cy.spec.js

```diff
@@ -90,4 +190,54 @@ describe("scenarios > dashboard > chained filter", () => {
       });
     });
   }
+
+  context("reproduces metabase#13868", () => {
+    it.only("can use a chained filter with embedded SQL questions", () => {
```

Comment:
Don't forget to change this to `.skip` :)

Reply:
Yep! I was kind of curious what impact `it.only` would have in CI, but looks like all the other tests still ran.

Reply:
That's expected - if we're talking about all OTHER tests/specs. `.only` has an effect for one specific test file... only :)
It will not prevent CI from running the other tests.